src/Item.cpp

```diff
@@ -0,0 +1,53 @@
+//
+// Created by Jahrr on 4/6/2023.
+//
+// Last edited by Jahrr on 4/6/2023
+//
+#include "Item.hpp"
+
+// Number values that are not set are set to -1 by default
+Item::Item() : name_{"Not Set"}, item_ID_{-1}, price_{-1}, stock_{-1} {}
+
+Item::Item(int item_ID, std::string item_name, float item_price, int item_stock)
+    : name_{std::move(item_name)}, price_{item_price}, stock_{item_stock} {}
+
+void Item::appendAdditionalInfo(const std::string& info) {
+  if (additional_info_.empty()) {
+    additional_info_ = info;
+  } else {
+    additional_info_ += ", " + info;
+  }
+}
+
+void Item::printItemInfo() {
+  // We have to check if the value has been set or not. Name is the exception
+  // because we can set it to "Not Set" by default.
+
+  // We need to remove the zeroes from this because std::to_string doesn't have
+  // formatting options
+  std::string price_string;
+
+  price_string = std::to_string(price_);
+  price_string.erase(price_string.find_last_not_of('0') + 1, std::string::npos);
+  price_string.erase(price_string.find_last_not_of('.') + 1, std::string::npos);
+
+  std::cout << "Item #" << item_ID_ << ": "
+            << "Name: " << name_ << ", "
+            << "Price: " << ((price_ >= 0.f) ? price_string : "Not Set") << ", "
+            << "Amount in Stock: "
+            << ((stock_ >= 0) ? std::to_string(stock_) : "Not Set") << ", "
+            << "Additional Information: "
+            << ((!additional_info_.empty()) ? additional_info_ : "None\n");
+}
```

Comment:
For printing the item info we should probably overload operator<< as a friend function something like this:
`
friend std::ostream& operator<<(std::ostream& os, const Item& item) {
         os << item.name_ << " " << item.item_ID_ << " " << item.price_ << " " << item.stock_;
         return os;
     }
`

Reply:
@Jahrr do we have an overload for the `<<` operator as @RockyDesigne suggested, or are we gonna leave it like it is?

Reply:
It seems unnecessary to overload the operator in this case. If we ever want to print to a file we can use std::stringstream. I'm going to leave it as is to keep it simple and understandable for an inexperienced reader, and if we need to we can change this in the future.